### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by ObsidianTTRPGProject**
Added new css style for Level formatting.

### CODE REVIEW

**Comment by mProjectsCode** [comment]
Maybe consider moving the images to a new line. This is how it looks on GitHub currently.
image

File: Pathfinder/2E/action-icons/readme.md [hunk 1]
```diff
@@ -23,16 +32,21 @@ It's made possible with a Javalent plugin who also deserves our thanks.
 1. [Markdown Attributes](https://github.com/valentine195/obsidian-markdown-attributes)
 2. Template (Core Plugins) or alternatively Templater can be used. 
 
-## Installation
-
+## Installation 
+### Action Icons
```
Comment (sigrunixia) [inline Pathfinder/2E/action-icons/readme.md:36]: Add a line Break before and after each header

```
## Installation

### Action Icons
```

File: Pathfinder/2E/action-icons/readme.md [hunk 2]
```diff
@@ -12,6 +12,15 @@ Ability to enter the Pathfinder 2e Action Icons into your notes.
 *⭓{ .Pathfinder }* - Free Action
```
Comment (sigrunixia) [inline Pathfinder/2E/action-icons/readme.md:12]: De-facto list items, so line break :)


```
*⭓{ .Pathfinder }* - Free Action

*⬲{ .Pathfinder }* - Reaction
```

File: Pathfinder/2E/action-icons/readme.md [hunk 3]
```diff
@@ -23,16 +32,21 @@ It's made possible with a Javalent plugin who also deserves our thanks.
...
+## Installation 
+### Action Icons
 1. Copy the \*.md from the Templates folder into your Vaults template folder. 
 2. Copy the '[PF2e-Actions-Font](_attachments/pf2e-actions-font.css)' into  your `.obsidian/snippets` folder. 
 3. Enable the snippet. `Settings > Appearance > CSS Snippets > PF2e Actions Font > Enable`
 4. Refer to the [Actions Cheat Sheet](notes/actions-cheat-sheet.md) and [Skill Actions Cheat Sheet](notes/skill-actions-cheat-sheet.md) to test out functionality of the icons.
 
-## Usage
+### Level Text Highlighter
+1. Copy the \*.md from the Templates folder into your Vaults template folder.
```
Comment (sigrunixia) [inline Pathfinder/2E/action-icons/readme.md:43]: Enforcement of space between Header and List Items

```
### Level Text Highlighter

 1. Copy the \*.md from the Templates folder into your Vaults template folder. 
```

File: Pathfinder/2E/action-icons/readme.md [hunk 4]
```diff
@@ -12,6 +12,15 @@ Ability to enter the Pathfinder 2e Action Icons into your notes.
...
 *⬲{ .Pathfinder }* - Reaction
 
+**Examples**
+![image](https://github.com/Obsidian-TTRPG-Community/ObsidianTTRPGShare/assets/96321415/6f4d5183-8592-4f5f-9372-2adc065657a0)
+
+Ability to format "Level" requirements similar to books. 
+
+Text{.hltr-level}
+**Example** 
+![image](https://github.com/Obsidian-TTRPG-Community/ObsidianTTRPGShare/assets/96321415/a2a60edf-4c32-4d9d-b836-7cb8310fee40)
```
Comment (sigrunixia) [inline Pathfinder/2E/action-icons/readme.md:22]: Github sees this as One Line.

```
Text{.hltr-level}

 **Example** 

 ![image](https://github.com/Obsidian-TTRPG-Community/ObsidianTTRPGShare/assets/96321415/a2a60edf-4c32-4d9d-b836-7cb8310fee40)
```

File: Pathfinder/2E/action-icons/readme.md [hunk 5]
```diff
@@ -12,6 +12,15 @@ Ability to enter the Pathfinder 2e Action Icons into your notes.
 *⭓{ .Pathfinder }* - Free Action
 *⬲{ .Pathfinder }* - Reaction
 
+**Examples**
+![image](https://github.com/Obsidian-TTRPG-Community/ObsidianTTRPGShare/assets/96321415/6f4d5183-8592-4f5f-9372-2adc065657a0)
```
Comment (sigrunixia) [inline Pathfinder/2E/action-icons/readme.md:16]: Similarly here, this is viewed as One line. Headers might also be appropriate here. Whichever ends up looking better.


```
**Examples**

 ![image](https://github.com/Obsidian-TTRPG-Community/ObsidianTTRPGShare/assets/96321415/6f4d5183-8592-4f5f-9372-2adc065657a0)
```

**Review by sigrunixia — CHANGES_REQUESTED** [review]

**Comment by sigrunixia** [comment]
Adding this here as well, if you use VSCode to check the work before adding it to Github, there is a lovely [plugin which shows a Github Preview](https://marketplace.visualstudio.com/items?itemName=bierner.github-markdown-preview). It _might_ also now be included in Github for VScode, but I do not use it anymore. 

Intellijidea and has it built in.

File: Pathfinder/2E/action-icons/readme.md (hunk 2 above)
Comment (sigrunixia) [reply Pathfinder/2E/action-icons/readme.md:15]: Resolved with my update

**Review by sigrunixia — APPROVED** [review]

**Pull request merged**